diff --git a/sdk/objc/components/audio/RTCAudioSessionConfiguration.m b/sdk/objc/components/audio/RTCAudioSessionConfiguration.m
index 74de6b5ce2..4eb54a62e0 100644
--- a/sdk/objc/components/audio/RTCAudioSessionConfiguration.m
+++ b/sdk/objc/components/audio/RTCAudioSessionConfiguration.m
@@ -60,7 +60,9 @@ static RTC_OBJC_TYPE(RTCAudioSessionConfiguration) *gWebRTCConfiguration = nil;
     // nonmixable, hence activating the session will interrupt any other
     // audio sessions which are also nonmixable.
     _category = AVAudioSessionCategoryPlayAndRecord;
+    #if !TARGET_OS_TV
     _categoryOptions = AVAudioSessionCategoryOptionAllowBluetooth;
+    #endif
 
     // Specify mode for two-way voice communication (e.g. VoIP).
     _mode = AVAudioSessionModeVoiceChat;
diff --git a/sdk/objc/components/capturer/RTCCameraVideoCapturer.h b/sdk/objc/components/capturer/RTCCameraVideoCapturer.h
index 5185fa866e..d5297983ba 100644
--- a/sdk/objc/components/capturer/RTCCameraVideoCapturer.h
+++ b/sdk/objc/components/capturer/RTCCameraVideoCapturer.h
@@ -19,6 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 // Camera capture that implements RTCVideoCapturer. Delivers frames to a
 // RTCVideoCapturerDelegate (usually RTCVideoSource).
 NS_EXTENSION_UNAVAILABLE_IOS("Camera not available in app extensions.")
+API_AVAILABLE(tvos(17.0), ios(12.0), macos(10.11))
 RTC_OBJC_EXPORT
 @interface RTC_OBJC_TYPE (RTCCameraVideoCapturer) : RTC_OBJC_TYPE(RTCVideoCapturer)
 
diff --git a/sdk/objc/components/capturer/RTCCameraVideoCapturer.m b/sdk/objc/components/capturer/RTCCameraVideoCapturer.m
index ff55bd7f19..bd4a0e2338 100644
--- a/sdk/objc/components/capturer/RTCCameraVideoCapturer.m
+++ b/sdk/objc/components/capturer/RTCCameraVideoCapturer.m
@@ -15,7 +15,7 @@
 #import "base/RTCVideoFrameBuffer.h"
 #import "components/video_frame_buffer/RTCCVPixelBuffer.h"
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
 #import "helpers/UIDevice+RTCDevice.h"
 #endif
 
@@ -42,7 +42,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
   FourCharCode _preferredOutputPixelFormat;
   FourCharCode _outputPixelFormat;
   RTCVideoRotation _rotation;
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
   UIDeviceOrientation _orientation;
   BOOL _generatingOrientationNotifications;
 #endif
@@ -80,7 +80,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
       return nil;
     }
     NSNotificationCenter *center = [NSNotificationCenter defaultCenter];
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
     _orientation = UIDeviceOrientationPortrait;
     _rotation = RTCVideoRotation_90;
     [center addObserver:self
@@ -171,7 +171,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
                                  format,
                                  (long)fps);
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
                       dispatch_async(dispatch_get_main_queue(), ^{
                         if (!self->_generatingOrientationNotifications) {
                           [[UIDevice currentDevice]
@@ -221,7 +221,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
                       }
                       [self.captureSession stopRunning];
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
                       dispatch_async(dispatch_get_main_queue(), ^{
                         if (self->_generatingOrientationNotifications) {
                           [[UIDevice currentDevice]
@@ -239,7 +239,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
 
 #pragma mark iOS notifications
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
 - (void)deviceOrientationDidChange:(NSNotification *)notification {
   [RTC_OBJC_TYPE(RTCDispatcher)
       dispatchAsyncOnType:RTCDispatcherTypeCaptureSession
@@ -267,7 +267,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
     return;
   }
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
   // Default to portrait orientation on iPhone.
   BOOL usingFrontCamera = NO;
   // Check the image's EXIF for the camera the image came from as the image
@@ -323,7 +323,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
 - (void)captureOutput:(AVCaptureOutput *)captureOutput
     didDropSampleBuffer:(CMSampleBufferRef)sampleBuffer
          fromConnection:(AVCaptureConnection *)connection {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
   CFStringRef droppedReason = CMGetAttachment(
       sampleBuffer, kCMSampleBufferAttachmentKey_DroppedFrameReason, nil);
 #else
@@ -338,7 +338,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
 
 - (void)handleCaptureSessionInterruption:(NSNotification *)notification {
   NSString *reasonString = nil;
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
   NSNumber *reason =
       notification.userInfo[AVCaptureSessionInterruptionReasonKey];
   if (reason) {
@@ -373,7 +373,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
   [RTC_OBJC_TYPE(RTCDispatcher)
       dispatchAsyncOnType:RTCDispatcherTypeCaptureSession
                     block:^{
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
                       if (error.code == AVErrorMediaServicesWereReset) {
                         [self handleNonFatalError];
                       } else {
@@ -427,7 +427,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
                     }];
 }
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
 
 #pragma mark - UIApplication notifications
 
@@ -568,7 +568,7 @@ const int64_t kNanosecondsPerSecond = 1000000000;
   NSAssert([RTC_OBJC_TYPE(RTCDispatcher)
                isOnQueueForType:RTCDispatcherTypeCaptureSession],
            @"updateOrientation must be called on the capture queue.");
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
   _orientation = [UIDevice currentDevice].orientation;
 #endif
 }
diff --git a/sdk/objc/components/renderer/metal/RTCMTLRenderer.h b/sdk/objc/components/renderer/metal/RTCMTLRenderer.h
index 84aa3fe7fc..3a56774799 100644
--- a/sdk/objc/components/renderer/metal/RTCMTLRenderer.h
+++ b/sdk/objc/components/renderer/metal/RTCMTLRenderer.h
@@ -37,7 +37,7 @@ NS_ASSUME_NONNULL_BEGIN
  * for performing cleanups.
  */
 
-#if TARGET_OS_IOS
+#if TARGET_OS_IOS || TARGET_OS_TV
 - (BOOL)addRenderingDestination:(__kindof UIView *)view;
 #else
 - (BOOL)addRenderingDestination:(__kindof NSView *)view;
diff --git a/sdk/objc/helpers/RTCCameraPreviewView.h b/sdk/objc/helpers/RTCCameraPreviewView.h
index 788e8c3e3e..3db184fdbb 100644
--- a/sdk/objc/helpers/RTCCameraPreviewView.h
+++ b/sdk/objc/helpers/RTCCameraPreviewView.h
@@ -18,6 +18,7 @@
 /** RTCCameraPreviewView is a view that renders local video from an
  *  AVCaptureSession.
  */
+API_AVAILABLE(tvos(17.0), ios(12.0), macos(10.11))
 RTC_OBJC_EXPORT
 @interface RTC_OBJC_TYPE (RTCCameraPreviewView) : UIView
 
diff --git a/sdk/objc/helpers/RTCCameraPreviewView.m b/sdk/objc/helpers/RTCCameraPreviewView.m
index 92c3cd2635..8d656ac825 100644
--- a/sdk/objc/helpers/RTCCameraPreviewView.m
+++ b/sdk/objc/helpers/RTCCameraPreviewView.m
@@ -81,6 +81,7 @@
 
 - (void)setCorrectVideoOrientation {
   // Get current device orientation.
+  #if TARGET_OS_IPHONE && !TARGET_OS_TV
   UIDeviceOrientation deviceOrientation = [UIDevice currentDevice].orientation;
   AVCaptureVideoPreviewLayer *previewLayer = [self previewLayer];
 
@@ -103,23 +104,28 @@
     // If device orientation switches to FaceUp or FaceDown, don't change video
     // orientation.
   }
+  #endif
 }
 
 #pragma mark - Private
 
 - (void)addOrientationObserver {
+  #if TARGET_OS_IPHONE && !TARGET_OS_TV
   [[NSNotificationCenter defaultCenter]
       addObserver:self
          selector:@selector(orientationChanged:)
              name:UIDeviceOrientationDidChangeNotification
            object:nil];
+  #endif
 }
 
 - (void)removeOrientationObserver {
+  #if TARGET_OS_IPHONE && !TARGET_OS_TV
   [[NSNotificationCenter defaultCenter]
       removeObserver:self
                 name:UIDeviceOrientationDidChangeNotification
               object:nil];
+  #endif
 }
 
 - (AVCaptureVideoPreviewLayer *)previewLayer {
